dev/cognitiveservices/data-plane/Language/analyzetext.json

```diff
@@ -1824,20 +2208,45 @@
         }
       }
     },
-    "EntityWithResolution": {
-      "type": "object",
+    "EntityWithTags": {
       "allOf": [
         {
           "$ref": "#/definitions/Entity"
+        },
+        {
+          "type": "object",
+          "required": [
+            "type",
+            "tags"
+          ],
+          "properties": {
+            "type": {
```

Comment:
During the review with the REST API Stewardship Board, it was suggested that the "category" and "subcategory" properties would be replaced by the new "type" and "tags" properties. I see in this PR that the latter are being added while the former are still there. Is this intentional? If so, how does it work? Are all four properties returned in the response?

Reply:
Yes, during the breaking change meeting it was discussed not removing them and still adding the types and tags to remove the breaking changes.  Currently subcategory is optional so we will not populate it and instead use the type to define the lowest granular entity.  The category will show the highest-level entity. Tags will be used to show all the entity types that could be assigned to the text.

Reply:
Thank you for clarifying. I see that custom NER still uses the same model (without the "type" and "tags" properties). Is the expectation to align as part of a separate PR?

Reply:
@joseharriaga Do I understand it correctly that the ask here is to update https://github.com/Azure/azure-rest-api-specs/blob/894688939e2624e292076071ec7e427b7dac5f6c/dev/cognitiveservices/data-plane/Language/analyzetext.json#L664 to instead use https://github.com/Azure/azure-rest-api-specs/blob/894688939e2624e292076071ec7e427b7dac5f6c/dev/cognitiveservices/data-plane/Language/analyzetext.json#L2135

@quentinRobinson could we do that and delete `EntitiesDocumentResult`?